internal/uvm/timezone.go

```diff
@@ -0,0 +1,78 @@
+package uvm
+
+import (
+	"fmt"
+
+	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
+	"golang.org/x/sys/windows"
+)
+
+var gmtTimezone = &hcsschema.TimeZoneInformation{
+	Bias:         0,
+	StandardName: "GMT Standard Time",
+	StandardBias: 0,
+	DaylightName: "GMT Daylight Time",
+	DaylightBias: -60,
+	StandardDate: &hcsschema.SystemTime{
+		Year:         0,
+		Month:        10,
+		DayOfWeek:    0,
+		Day:          5,
+		Hour:         2,
+		Minute:       0,
+		Second:       0,
+		Milliseconds: 0,
+	},
+	DaylightDate: &hcsschema.SystemTime{
+		Year:         0,
+		Month:        3,
+		DayOfWeek:    0,
+		Day:          5,
+		Hour:         1,
+		Minute:       0,
+		Second:       0,
+		Milliseconds: 0,
```

Comment:
Are these just dummy values?

Reply:
No, it's the actual values for the time zone (unless I mistyped somewhere).

Reply:
Oh okay, I'm not super familiar with timezones